doc/source/ray-observability/monitoring-debugging/profiling.rst

```diff
@@ -3,6 +3,78 @@
 Profiling
 =========
 
+.. _ray-core-mem-profiling:
+
+Memory profile Ray Actors and Tasks
+-----------------------------------
+
+To memory profile Ray tasks or actors `memray <https://bloomberg.github.io/memray/>`_.
+Note that you can also use other memory profiling tools if it supports a similar API.
+
+First, download ``memray``.
+
+.. code-block:: bash
+
+  pip install memray
+
+``memray`` supports a Python context manager to enable memory profiling. You can write the ``memray`` profiling file wherever you want.
+But in this example, we will write them to `/tmp/ray/session_latest/logs` because Ray dashboard allows you to download files inside a log file.
```

Comment:
```suggestion
But in this example, we will write them to `/tmp/ray/session_latest/logs` because Ray dashboard allows you to download files inside the log folder.
```